frontend/src/components/CanteenIndicators.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <div v-if="canteen.dailyMealCount">
+      <v-icon small>mdi-silverware-fork-knife</v-icon>
+      {{ canteen.dailyMealCount }} repas par jour
+    </div>
+    <div v-if="canteen.city">
+      <v-icon small>mdi-compass</v-icon>
+      {{ canteen.city }}
+    </div>
+    <div v-if="sectors">
+      <v-icon small>mdi-office-building</v-icon>
+      {{ sectors }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "CanteenIndicators",
+  props: {
+    canteen: {
+      type: Object,
+    },
+  },
+  computed: {
+    sectors() {
+      if (!this.canteen || !this.canteen.sectors) return null
```

Comment:
```suggestion
      if (!this.canteen.sectors) return null
```
`this.canteen` est obligatoire dans les props ici je pense ?

Reply:
J'ai profité pour ajouter le `required: true` dans le param : https://github.com/betagouv/ma-cantine/pull/466/commits/59cda0a124bcf05107f0cb5ece303cd831f8eb7b